x-pack/plugins/security_solution/public/management/components/endpoint_action_failure_message/endpoint_action_failure_message.tsx

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
FYI. I created this component under `management/components/` because it was going to be used in the Action Log as well, but found out later that the Action list API does not return the action's output (by design), thus we can't really use it just yet.